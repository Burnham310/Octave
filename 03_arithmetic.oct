main = Section(Int) ;octave=5; {
    1+5 5-3 2*3+1
}
