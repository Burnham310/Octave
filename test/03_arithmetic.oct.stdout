[
{
  "freq": 219.9953155517578,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 146.82925415039062,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 246.93638610839844,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0,
  "duration": 0,
  "gap": 0,
  "amp": 0,
  "instrument": "dummy"
}

]